xc/device.ml

```diff
@@ -1451,6 +1451,35 @@ module Vusb = struct
       in
       exec_usb_reset_script argv
 
+  let qom_list ~xs ~domid =
+    (*. 1. The QEMU Object Model(qom) provides a framework for registering user
+         creatable types and instantiating objects from those types.
+        2. Qom types can be instantiated and configured directly from the QEMU
+         monitor or command-line (eg,-device, device_add).
+        3. Command example:
+         {"execute":"qom-list","arguments":{"path":"/machine/peripheral"}}
+         result:
+         {"return": [{"name": "usb1", "type": "child<usb-host>"}, {"name":"type", "type": "string"}}
+         The usb1 is added.
+    *)
+    debug "qom list to check if usb device is attached to vm";
```

Comment:
This debug message need more context to be useful in the logs; it needs at least the domid. I would also move it further down and include the path because the qmp command might not be executed when `is_running` is false. However, `qmp_send_cmd` itself logs all commands - so this debug  message could be removed entirely as it does not add new information.

Reply:
Ok. Will remove that.

Reply:
Updated.